core/rest/Makefile

```diff
@@ -35,6 +36,8 @@ build_all:
 	echo "$(MAVEN) site package -DskipTests"
 	echo "### For full Maven output, see file build_rest.log"
 	set -o pipefail && $(MAVEN) site package -DskipTests | tee build_rest.log | grep --line-buffered -E -e '^\[[^WId]' -e '^\[INFO\] B[Uu][Ii][Ll][Dd]' -e 'to compile'
+	mkdir -p ../../${DISTRIBUTION_DIR}
+	cp -pf target/$(BLD_TRAFODION_REST_TARNAME) ../../${DISTRIBUTION_DIR}/
```

Comment:
Same issue here. Copy leaves file in both old and new location. Would prefer one location or the other.